list-unicode-display.el

```diff
@@ -34,6 +34,12 @@
 (define-derived-mode list-unicode-display-mode help-mode "Unicode Characters"
   "Major mode to display a list of unicode characters.")
 
+(defface list-unicode-display-code-point
+  '((t :foreground "light green"
```

Comment:
Here it would be better to inherit an existing face, so that it doesn't immediately require changes to every theme in order to be legible.